Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -979,7 +1003,8 @@ namespace AzToolsFramework
                     }
                     if (isFolder)
                     {
-                        AZ::IO::SystemFile::DeleteDir(fromPath.substr(0, fromPath.size() - 2).data());
+                        AZStd::string fromPathNoWildcard(fromPath.substr(0, fromPath.size() - 2));
+                        AZ::IO::SystemFile::DeleteDir(fromPathNoWildcard.c_str());
                     }
```

Comment:
@AMZN-stankowi FYI I added this fix because the folder wasn't getting deleted.

Reply:
Looking at SystemFile_WinAPI.cpp, it doesn't print out an error or anything if this call fails. Would it make sense to print out a message?

```
if(!AZ::IO::SystemFile::DeleteDir(fromPathNoWildcard.c_str()))
{
    AZ_Printf("AssetBrowser", false, "Failed to delete empty directory %s.", fromPathNoWildcard.c_str());
}
```

No problem if not, it doesn't look like many other places that call AZ::IO::SystemFile::DeleteDir are checking, either.

Reply:
Makes sense. Added as a warning.